apps/core/declarativewebcontainer.h

```diff
@@ -139,6 +139,7 @@ class DeclarativeWebContainer : public QWindow, public QQmlParserStatus, protect
     bool isActiveTab(int tabId);
     bool activatePage(const Tab& tab, bool force = false);
     int findTabId(uint32_t uniqueId) const;
+    int tabUsedBefore() const;
```

Comment:
Perhaps there is some ambiguity here (it could mean "was this tab used before?"). There also seems to be a common pattern to have the verb at the front and the subject after. Maybe `previouslyUsedTab` or `previouslyActiveTab`?

Reply:
Both `previouslyUsedTab` and `previouslyActiveTab` sound better.

Reply:
Fixed. Went with `previouslyUsedTabId`